ansible/roles/mixing/tasks/main.yml

```diff
@@ -0,0 +1,62 @@
+---
+
+# Mixing
+
+- name: Ensure jq is installed
+  ansible.builtin.apt:
+    name: jq
+    state: present
+    update_cache: true
+  become: true
+
+- name: Create CoinJoin wallets
+  ansible.builtin.command:
+    cmd: dash-cli createwallet {{ coinjoin_wallet_name }} false false "" false true
```

Comment:
This causes infinite recursion of the following message:
```
fatal: [mixer-1]: FAILED! => 
  msg: 'An unhandled exception occurred while templating ''{{ coinjoin_wallet_name }}''. Error was a <class ''ansible.errors.AnsibleError''>, original message: An unhandled exception occurred while templating ''{{ coinjoin_wallet_name }}''.
```

Reply:
Is that infinite, or do you just not have coinjoin_wallet_name in your .yml file? Can you check?